indico/modules/events/editing/service.py

```diff
@@ -123,3 +124,27 @@ def service_get_status(event):
     except requests.RequestException as exc:
         return {'status': None, 'error': unicode(ServiceRequestFailed(exc))}
     return {'status': resp.json(), 'error': None}
+
+
+def service_handle_editable(editable):
```

Comment:
`service_handle_editable_submission` or `service_handle_new_editable`?

Reply:
I like the second one.